tox.ini

```diff
@@ -7,7 +7,7 @@ envlist = flake8,ansible-lint
 basepython = python2.7
 deps =
     ansible>=1.9.1,<2.0.0
-    -ropenstack-ansible/test-requirements.txt
+    -rtest-requirements.txt
```

Comment:
This is probably a hold over from the portion of the original PR that also removed OSA as a submodule. It's almost certainly not necessary now but probably not a harmful bit of future-proofing since we will need this *when* we remove the submodule.